App/src/services/user.ts

```diff
@@ -4,11 +4,18 @@ export interface User {
     email: string | null;
```

Comment:
You can remove the `| null` from all the fields in User, and also add an `id: string;` field

Reply:
done